uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
We'll want a similar 2D barcode for groups in the future, so I figured I'd add the scaffolding now to make our lives easier.